db/migration/V016__add_category_id_to_competencies.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE competencies
+  ADD COLUMN category_id BIGINT,
```

Comment:
Do we want to add an `INDEX` here as well? I noticed that other tables that have foreign keys have done this (such as in the V_011 and V_006 migration files). I believe it is supposed to increase the speed of querying the table, but I'm not sure if it is necessary or not.

Reply:
Good Call @sheacronin ! @denizedaza Let's add it in to be consistent before merge.

Reply:
On it!

Reply:
If I'm understanding the `INDEX` property correctly, would it be something like this?
<img width="577" alt="image" src="https://user-images.githubusercontent.com/10994868/175659300-44e727fa-56f2-4518-89f5-b8f49a2552d4.png">

Reply:
Yes that fits the bill!